docs/basics/server.md

```diff
@@ -290,7 +290,7 @@ curl -XPOST -Hcontent-type:application/json -d *{"gremlin":"g.V().count()"}* [IP
 
 ## JanusGraph Server as Both a WebSocket and HTTP Endpoint
 
-As of JanusGraph 0.2.0, you can configure your `gremlin-server.yaml` to
+As of JanusGraph 0.2.0, you can configure your `janusgraph-server.yaml` to
```

Comment:
As I understand as of 0.2.0 it is still `gremlin-server.yaml` but as for `0.6.0` it is `janusgraph-server.yaml`

Reply:
I revert this change.